neoload/commands/project.py

```diff
@@ -7,17 +7,24 @@
 @click.argument("command", required=True, type=click.Choice(['up', 'upload', 'meta']))
 @click.option("--path", "-p", type=click.Path(exists=True), default='.',
               help="path of project folder, zip or yml file. . is default value")
+@click.option("--display-progress/--just-json", required=False, default=True,
```

Comment:
The default value will be discussed in the meeting of this afternoon.

Reply:
There is a check, regardless of if the default is True, that if the execution context is not in interactive mode, it will surpress progress (non-JSON output), so is compatible with CI and testing context but also with interactive user by default when file is less than 100KB upload.